### DISCUSSION TITLE
Add more different tags

### DISCUSSION THREAD

**Discussion by ChillLP** [Traewelling/traewelling#2067] [Ideas / Feature Requests]
Regarding my Issue on the Träweldroid repo; https://github.com/Traewelldroid/traewelldroid/issues/256

What i thought of

-  ```trwl:route_name``` For routes with names (IC2214 "Deichgraf")

- ```trwl:locomotive_name``` or ```trwl:vehicle_name``` For locomotives or rather consists with a Name, example "Treuchtlingen"

-  ```trwl:locomotive_number``` For Just the specific numbering of the locomotive or consist, example (101) "139"

**Comment by marhei** [comment]
I add: `trwl:role` to indicated the role on the train (Tf, Zf, Gf, …)

**Reply by ChillLP** [reply]
I Just wanted to add that xD

**Reply by MrKrisKrisu** [reply]
`trwl:role` was added to the documentation in #2095.

**Comment by marhei** [comment]
I would also suggest `trwl:route` to indicate the real route in case of a detour.

**Reply by MrKrisKrisu** [reply]
Can you give an example usage of this? I'm not sure if I've understood that correctly.

**Reply by marhei** [reply]
As it is well known that the polyline generated by Hafas or Brouter is not always correct or up-to-date in the event of a (short-term) diversion, it would be nice to be able to record the actual route in a tag.

**Comment by MrKrisKrisu** [comment]
I'm still missing a key for the vehicle number. I've often seen `trwl:locomotive_class` being misused for this.

I suggest: `trwl:vehicle_number`

**Reply by ChillLP** [reply]
i thought of `trwl:locomotive_number ` - but vehicle is better in that case yeah

**Reply by MrKrisKrisu** [reply]
added in #2133

**Comment by ChillLP** [comment]
Just crossed my mind, a Tag for a "Tz Number" would be nice
`trwl:mu_number` 
or alike

**Reply by MrKrisKrisu** [reply]
Wouldn't `trwl:vehicle_number` also be suitable? The proposal is deliberately kept very general.

**Comment by dasBunny** [comment]
As I usually use Träwelling to figure out what trains I took for passenger rights requests, a tag where I can note down the kind of claim and whether it has been processed would be helpful. Maybe `trwl:passenger_rights`

**Reply by MrKrisKrisu** [reply]
added to default tags in #2175

**Comment by pascalwittler** [comment]
A tag for the seat capacity on the train (and/or the coach) would be nice for others as a more accurate "real time" indicator compared to the prediction provided by DB Fernverkehr.

**Comment by ET430-Rosenheim** [comment]
Would `trwl:operator` make sense given that many ÖPNV are simply marked as "Nahreisezug" in HAFAS and therefore Träwelling doesn't recognize the individual operators

**Reply by MrKrisKrisu** [reply]
As the "Operator" data model already exists separately for this, I would not use this as an extra tag. Instead, we should work towards ensuring that incorrect data can be corrected.

However, you can still use any keys in your status tags.
